src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -221,6 +222,29 @@
                          :customization-color account-color
                          :on-press            on-press}}]))
 
+(defn- pay-token-bottom-sheet
+  []
+  [select-asset/view
+   {:title            (i18n/label :t/select-asset-to-pay)
+    :on-select        (fn [token]
+                        (rf/dispatch [:wallet.swap/select-asset-to-pay {:token token}]))
+    :hide-token-fn    (fn [type _]
+                        (= type constants/swap-tokens-popular))
+    :disable-token-fn (fn [type {:keys [available-balance]}]
+                        (and (= type constants/swap-tokens-my)
+                             (= available-balance 0)))}])
+
+(defn- receive-token-bottom-sheet
```

Comment:
I'd rather use a modal with sheet presentation instead of a bottom sheet. Check the implementation of set spending cap screen.

https://github.com/status-im/status-mobile/blob/0eb065bbb723be6a235d41b14fb28bf2a65a9ba2/src/status_im/navigation/screens.cljs#L543-L545

https://github.com/status-im/status-mobile/blob/052e412c60efa80e062e747a61d2e2454070882d/src/status_im/contexts/wallet/swap/setup_swap/view.cljs#L383